main.go

```diff
@@ -31,30 +31,29 @@ func createRepo(name, description interface{}) error {
 	return repo.CreateRepo(repoName, repoDesc)
 }
 
-func upload(path interface{}) error {
-	// Check if the current directory is a git repository.
-	// If it has any unstaged commits it should stage, commit, and push.
-	// repo.UploadRepo(path)
-	return fmt.Errorf("Command [upload] not yet implemented.")
+func upload(patharg interface{}) error {
+	var pathstr string
+	if patharg == nil {
+		pathstr = ""
```

Comment:
I think this could be skipped, since line 35 already initializes `pathstr` as `""`.